src/v3/src/components/PasswordWithConfirmation/PasswordWithConfirmation.tsx

```diff
@@ -59,6 +61,9 @@ const PasswordWithConfirmation: UISchemaElementComponent<{
       <Box>
         <PasswordInput
           label={label}
+          tooltipLabel={
+            (isHidden: boolean) => (isHidden ? getTranslation(tooltipTranslations, 'show') : getTranslation(tooltipTranslations, 'hide'))
+          }
```

Comment:
heads up this will be a merge conflict with @glenfannin-okta PR since his PR refactors and subsequently reuses the `InputPassword` element to replace this `PasswordInput`.